corehq/form_processor/backends/sql/update_strategy.py

```diff
@@ -302,3 +349,56 @@ def _apply_form_transaction(self, transaction):
             filtered_updates = [u for u in case_updates if u.id == self.case.case_id]
             for case_update in filtered_updates:
                 self._apply_case_update(case_update, form)
+
+
+def _transaction_sort_key_function(case):
+    xform_ids = list(case.xform_ids)
```

Comment:
Is this cached / already populated? Want to make sure it's not doing another db lookup.

Reply:
Yeah, it's [memoized](https://github.com/dimagi/commcare-hq/blob/d6018d3796bebdd91208af63f4cb7b7d39f50c71/corehq/form_processor/models.py#L691), as is [transactions](https://github.com/dimagi/commcare-hq/blob/d6018d3796bebdd91208af63f4cb7b7d39f50c71/corehq/form_processor/models.py#L801), so if either has already been called (and transactions should have been by this point), it shouldn't require another db call.